### PR TITLE
chore: fix the source value error in web-types

### PR DESCRIPTION
**Pull request by tolking**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

fix the source value error in web-types, ensure that in theory you can jump to the type definition files by the components name in the webstorm. 

```diff
- "source": {"symbol": "Affix"},
+ "source": {"symbol": "ElAffix"},
```

### CODE REVIEW

File: build/helper.ts [hunk 1]
```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `EL${title.replace(/-/, ' ').replace(/^\w|\s+\w/g, (item) => {
```
Comment (JeremyWuuuuu) [inline build/helper.ts:23]: Should the symbol be started with `El`? I think `EL` seems wrong to me.

Comment (tolking) [reply build/helper.ts:23]: Yes this is a typo

File: build/helper.ts [hunk 2]
```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `El${title.replace(/-/, ' ').replace(/^\w|\s+\w/g, (item) => {
```
Comment (sxzz) [inline build/helper.ts:23]:
```suggestion
  const symbol = `El${title.replace('-', ' ').replaceAll(/^\w|\s+\w/, (item) => {
```

Comment (tolking) [reply build/helper.ts:23]: When using regular, the replace is the same as the replaceAll. For easy reading?

Comment (sxzz) [reply build/helper.ts]: Yeah. I think `replaceAll` is better.

File: build/helper.ts [hunk 3]
```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `El${title.replace(/-/g, ' ').replace(/^\w|\s+\w/g, (item) => {
```
Comment (sxzz) [inline build/helper.ts:23]:
```suggestion
  const symbol = `El${title.replaceAll('-', ' ').replaceAll(/^\w|\s+\w/, (item) => {
```

Comment (tolking) [reply build/helper.ts:23]:
```js
.replaceAll(/^\w|\s+\w/,
```

Is the wrong way to use, it should be

```js
.replaceAll(/^\w|\s+\w/g,
```

It's the same as replace

Comment (sxzz) [reply build/helper.ts:23]: My mistake. You're right.

**Review by sxzz — APPROVED** [review]

**Pull request merged**